### ISSUE TITLE
Fix makefile

### ISSUE THREAD

**Issue by Mortinat** [UrLab/DocHub#263]
il faudrait fix le makefile qui load les cours sans avoir cree le tree avant

**Comment by C4ptainCrunch** [comment]
Indeed, le makefile appelle `./manage.py shell < ./catalog/management/commands/load_courses.py` mais ne crée pas de tree avant, ce qui fait que http://127.0.0.1:8000/catalog/ crash et renvoie une 404 et qu'on ne peut pas utiliser le finder.
